app/models/client.rb

```diff
@@ -35,6 +35,7 @@ def self.create_from_intake(intake)
       phone_number: intake.phone_number,
       sms_phone_number: intake.sms_phone_number,
       vita_partner: intake.vita_partner,
+      documents: intake.documents,
```

Comment:
This should copy the documents from the intake onto the client directly, which should work for now.